sdk/communication/communication-common/CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Release History
 
-## 1.2.0 (Unreleased)
+## 2.0.0 (Unreleased)
 
 ### Features Added
 
 - Optimization added: When the proactive refreshing is enabled and the token refresher fails to provide a token that's not about to expire soon, the subsequent refresh attempts will be scheduled for when the token reaches half of its remaining lifetime until a token with long enough validity (>10 minutes) is obtained.
+- Migrated from using `@azure/core-http` to `@azure/core-rest-pipeline` for the handling of HTTP requests. See [Azure Core v1 vs v2](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/core-rest-pipeline/documentation/core2.md) for more on the difference and benefits of the move.
```

Comment:
This should go under the breaking changes section with a note on the policies/classes being dropped/replaced

Reply:
thank you, fixed